### PR TITLE
Oy2 9926 - Measure 40: DEV-CH

### PR DESCRIPTION
**Pull request by jdinh8124**
## Purpose
Measure 40: DEV-CH

#### Linked Issues to Close
https://qmacbis.atlassian.net/jira/software/c/projects/OY2/boards/208?modal=detail&selectedIssue=OY2-9926&assignee=616db787702bd0006a4cfda8&assignee=61926b58657a0500700d1f4a

#### Pull Request Creator Checklist

- [x] This PR has an associated issue or issues.
- [x] The associated issue(s) are linked above.
- [x] This PR meets all acceptance criteria for those issues.
- [x] This PR and linked issue(s) are adequately documented
- [x] This PR and linked issues(s) are a complete description of the changeset; an individual or team should be able to understand the issue(s) and changes by reading through this PR and it's links, with no further interaction.
- [x] Someone has been assigned this PR.
- [x] At least one person has been marked as reviewer on this PR.

#### Pull Request Reviewer/Assignee Checklist

- [ ] This PR has an associated issue or issues.
- [ ] The associated issue(s) are linked above.
- [ ] This PR meets all acceptance criteria for those issues.
- [ ] This PR and linked issue(s) are adequately documented
- [ ] This PR and linked issues(s) are a complete description of the changeset; an individual or team should be able to understand the issue(s) and changes by reading through this PR and it's links, with no further interaction.

### CODE REVIEW

**Comment by abelzer212** [comment]
Screen Shot 2022-03-23 at 1 54 06 PM
In Deviation from performance measure it should say "Children Total" Not "Total"

**Comment by abelzer212** [comment]
Screen Shot 2022-03-23 at 1 57 21 PM
Screen Shot 2022-03-23 at 1 57 29 PM

If No data source is selected, Rate should be editable

File: services/ui-src/src/dataConstants.ts [hunk 1]
```diff
@@ -83,8 +83,8 @@ export const DEVIATION_OPTIONS = "DeviationOptions";
 export const DID_CALCS_DEVIATE = "DidCalculationsDeviate";
 export const DID_COLLECT = "DidCollect";
 export const DID_REPORT = "DidReport";
-export const END_DATE = "endDate";
 export const EHR_DATA = "Electronic Health Record (EHR) Data";
+export const END_DATE = "endDate";
```
Comment (abelzer212) [inline services/ui-src/src/dataConstants.ts:87]: I love to see alphabetizing

**Review by abelzer212 — APPROVED** [review]

**Review by abelzer212 — APPROVED** [review]

**Review by benjaminpaige — APPROVED** [review]
looks good!

**Pull request merged**